### PR TITLE
Update episode 05 checklists

### PR DESCRIPTION
**Pull request by SarahAlidoost**

### CODE REVIEW

**Comment by mkuzak** [comment]
thank you @SarahAlidoost I'll have a look at it now 👍

File: _episodes/05-checklists.md [hunk 1]
```diff
@@ -8,44 +8,58 @@ questions:
...
+
+> ## [Recommendation 5](https://fair-software.eu/recommendations/checklist)
+>
 > Use a software quality checklist
 {: .callout}
+![recommendation5]({{ page.root }}/fig/recommendation5.png)
+
+## Why software checklists are important
+
+Checklists help you write good quality software. What exactly **constitutes
```
Comment (mkuzak) [inline _episodes/05-checklists.md]:
```suggestion
Checklists help you write good quality software. What exactly
```

File: _episodes/05-checklists.md [hunk 2]
```diff
@@ -8,44 +8,58 @@ questions:
...
+good quality** means depends on the specific application of the software,
```
Comment (mkuzak) [inline _episodes/05-checklists.md]:
```suggestion
**good quality** means depends on the specific application of the software,
```

**Review by mkuzak — CHANGES_REQUESTED** [review]

**Comment by mkuzak** [comment]
only few tiny changes to apply before merging

**Comment by SarahAlidoost** [comment]
> only few tiny changes to apply before merging

Done! :-)

**Comment by mkuzak** [comment]
Merged 🚀
Great work, thank you so much. I guess we're ready to start the weekend.

**Pull request merged**